oauth2_http/java/com/google/auth/oauth2/ExternalAccountCredentials.java

```diff
@@ -415,10 +411,11 @@ static ExternalAccountCredentials fromJson(
     String clientSecret = (String) json.get("client_secret");
     String quotaProjectId = (String) json.get("quota_project_id");
     String userProject = (String) json.get("workforce_pool_user_project");
-    String universeDomain = (String) json.get("universe_domain");
     Map<String, Object> impersonationOptionsMap =
         (Map<String, Object>) json.get("service_account_impersonation");
 
+    GoogleCredentials baseCredential = GoogleCredentials.fromJson(json);
```

Comment:
Is there no way to make this a super() call or something? Seems awkward to build a seperate credential here just to get the universe domain.

Reply:
Using GoogleCredentials as a concrete holder of JSON data is surprising.  It would be better to provide something like GoogleCredentialJson as a simple data class to wrap the contents of a data file, but it ought to fully specify the contents of the file parsed.

Lacking a full specification of the format shared by all credentials, it is better to inline reading of all values in each concrete credential.

Thanks!